rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -26,8 +26,8 @@ elasticsearch_apt_packages:
   - openjdk-7-jre-headless
 
 elasticsearch_pip_packages:
-  - elasticsearch<2.1.0
-  - elasticsearch-curator<3.5.0
+  - elasticsearch>=2.0.0,<3.0.0
+  - elasticsearch-curator==4.0.4
```

Comment:
Hm. We provide a range above and a pin here. Why are we pinning instead of providing a range with a bunch of versions to exclude?